app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -18,7 +18,7 @@ def authenticate
       @current_user = load_user_object
       validate_request_ip
       @current_user.present?
-    rescue Errors::AccessTokenExpiredError => e
+    rescue Errors::AccessTokenExpiredError, Errors::MPILockedAccountError => e
```

Comment:
I actually wondered if it'd be preferred for this to rescue in the `StandardError` flow? Did you see a difference in behavior with one versus the other?

Reply:
The thought was to have it have the `Forbidden` status code but the logging is more important.